_authors/matti-schneider.md

```diff
@@ -4,5 +4,5 @@ first_name: Matti
 last_name: Schneider
 full_name: Matti Schneider
 published: true
+status: nil
```

Comment:
how about:
```suggestion
active: false
```

Reply:
Looks good. Will change it for all :smile: